src/main/com/example012/v1/LotteryGameApplicationV1.java

```diff
@@ -0,0 +1,16 @@
+package main.com.example012.v1;
+
+public class LotteryGameApplicationV1 {
+
+    public static void main(String[] args) {
+
+        LotteryGame lotteryGame = new LotteryGame();
+        lotteryGame.lotteryGame(Company.SPORTKA, 100000);
+        lotteryGame.sortResults();
+        lotteryGame.printGameResults();
+        LotteryGame lotteryGame2 = new LotteryGame();
```

Comment:
před tou druhou **lotteryGame2** prázdný řádek, ať to je ucelené do nějakých logických celků